node/malus/integrationtests/0002-dispute-invalid-block.toml

```diff
@@ -0,0 +1,55 @@
+[settings]
+timeout = 1000
+
+[relaychain]
```

Comment:
Hi @drahnr / @ordian, I checked this configuration (using `polkadot-local` chain) and it's failing to register the `parachain` since the `sudo` isn't available. I checked `polkadot_local_testnet_genesis` fn and `Alice` and `Bob` are set as `initial_authtorities`

```
#[cfg(feature = "polkadot-native")]
fn polkadot_local_testnet_genesis(wasm_binary: &[u8]) -> polkadot::GenesisConfig {
	polkadot_testnet_genesis(
		wasm_binary,
		vec![
				get_authority_keys_from_seed_no_beefy("Alice"),
				get_authority_keys_from_seed_no_beefy("Bob"),
		],
		get_account_id_from_seed::<sr25519::Public>("Alice"),
		None,
	)
}
```

So, I try to set only `alice` and `bob` as validators, but again I don't see the  `sudo` when I connect to the `rpc`

![image](https://user-images.githubusercontent.com/363911/140501591-72a8cd2e-37c6-473a-88f7-ac7d4230e8a3.png)

Is something that I have miss-configured?

Thanks!!

Reply:
I think only 0001 scenario was tested properly. I don't know why we use `polkadot-local` here, I've tweaked `wococo-local` specifically for simnet to have 4 authorities (#3998) and `sudo`. 

Short-term I would suggest try making it work with wococo-local (with 4 authorities) and long-term we need to generate the spec ourselves to allow for arbitrary number of authorities (you can take a look at how polkadot-launch does that: https://github.com/paritytech/polkadot-launch/blob/master/src/spec.ts).